examples/timeline/interaction/eventListeners.html

```diff
@@ -83,11 +86,23 @@
       logEvent(event, properties);
     });
 
+    function elementsCensor (object) {
```

Comment:
Maybe this function is better called "stringifyObject" or something similar. "Censor" is realy not the right word here..